lib/appcues_flutter.dart

```diff
@@ -81,6 +84,66 @@ class AppcuesView extends SemanticsTag {
   const AppcuesView(this.identifier) : super(identifier);
 }
 
+class AppcuesFrameView extends StatefulWidget {
+  final String frameId;
+
+  const AppcuesFrameView(this.frameId, {Key? key}) : super(key: key);
+
+  @override
+  _AppcuesFrameViewState createState() => _AppcuesFrameViewState();
+}
+
+class _AppcuesFrameViewState extends State<AppcuesFrameView> {
+  // A non-zero height is needed to ensure that the native view
+  // layoutSubviews is called at least once. Then, the intrinsic size of
+  // the native view will control the SizedBox _height here to auto
+  // size contents or set to zero if hidden.
+  double _height = 0.1;
```

Comment:
A gotcha I ran into here, where sometimes a view would never get registered on the native side, even though it was added to the layout on the Flutter side.